src/Bicep.LangServer/Server.cs

```diff
@@ -142,6 +143,7 @@ private static void RegisterServices(CreationOptions creationOptions, IServiceCo
             services.AddSingleton<IDeploymentOperationsCache, DeploymentOperationsCache>();
             services.AddSingleton<IDeploymentFileCompilationCache, DeploymentFileCompilationCache>();
             services.AddSingleton<IClientCapabilitiesProvider, ClientCapabilitiesProvider>();
+            services.AddSingleton<ApiVersionProvider, ApiVersionProvider>();
```

Comment:
This should be .AddSingleton<IApiVersionProvider, ApiVersionProvider>()?

Reply:
Reverted, not working